diagnostic.md

```diff
@@ -21,13 +21,19 @@ Write the `curl` request you'd use to retrieve a list of all donuts on the
 server.
 
 ```sh
-# your answer here
+
+curl --request GET http://www.example.com/donuts
 ```
 
 Write an AJAX request to retrieve a list of all donuts on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = function () {
+    $.ajax({
```

Comment:
it's always good to get in a habit of preventing default actions by passing in the event, then preventing the default action on it.

something like this:
```
let getDonuts = function (event) {
  event.preventDefault();
```